activestorage/lib/active_storage/engine.rb

```diff
@@ -154,6 +154,18 @@ class Engine < Rails::Engine # :nodoc:
       end
     end
 
+    initializer "action_view.configuration" do
+      config.after_initialize do |app|
+        ActiveSupport.on_load(:action_view) do
+          multiple_file_field_include_hidden = app.config.active_storage.delete(:multiple_file_field_include_hidden)
```

Comment:
@seanpdoyle out of curiosity why does this config live on Active Storage and not on Action View? It doesn't seem to have any real relationship to Active Storage, and it impacts Action View even if AS is not being used (but is loaded).